### DISCUSSION TITLE
Github desktop seems to have broken on Mac

### DISCUSSION THREAD

**Discussion by leftCoast** [community/community#17646] [Projects and Issues]
Have a iMac with 10.13.6 and suddenly my github desktop won't start up anymore.  Was this an update that killed this? Any hints on what I can try to make this work again? I'm kinda' running in the dark here.

Thanks!

-jim lee

**Comment by steveward** [comment]
Sorry for the trouble! This should be fixed in the latest release of GitHub Desktop. Let us know if you're still having issues.
